spec/game_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'game'
+RSpec.describe Game do 
+	subject(:r_vs_p){	described_class.new('Rock','Paper')}
+	subject(:p_vs_s){	described_class.new('Paper','Scissors')}
+	subject(:s_vs_r){	described_class.new('Scissors','Rock')}
+	
+	describe '#initialize' do 
+		it 'has player option' do 
+			expect(r_vs_p.player_option).to eq 'Rock'
+		end 
+
+		it 'has computer option' do
+			expect(p_vs_s.computer_option).to eq 'Scissors'
+		end 
+	end 
+
+	describe '#win?' do 
+		it 'let rock defeated by paper' do
+			r_vs_p.win?
+			expect($win).to eq false
```

Comment:
I don't think you need to use a global variable here either
the method `win?` could return a boolean and then the test would be:
```
has_won = r_vs_p.win?
expect(has_won).to eq false
```

Reply:
Though since you are checking for the draw state too it might be better not to use a boolean here and instead return a string explaining what happened